sbin/anti-evil-maid-lib

```diff
@@ -87,13 +90,24 @@ waitforenter() {
     fi
 }
 
+tpmfail() {
+    message "Unexpected TPM version: $TPM_VERSION"
+    return 1
+}
+
 checktpmnvram() {
     # checks whether the TPM NVRAM area is defined
-    # NOTE: tpm_nvinfo does not return non-zero if requested index
-    # is not a defined NVRAM area so we need to parse
-    if ! tpm_nvinfo -i "$TPM_FRESHNESS_INDEX" | grep -q 'AUTHWRITE'; then
-        return 1
-    fi
+    case "$TPM_VERSION" in
+        # NOTE: tpm_nvinfo does not return non-zero if requested index
+        # is not a defined NVRAM area so we need to parse
+        1) tpm_nvinfo -i "$TPM_FRESHNESS_INDEX" | grep -q 'AUTHWRITE' ;;
+        # NOTE: tpm2_readpublic return all defined NV indices so we need to
+        # parse
+        2) tpm2_readpublic |
```

Comment:
I think this should be `tpm2_nvreadpublic`, no?

Reply:
It should, thanks.